pkg/apiserver/rest/dualwriter_mode1_test.go

```diff
@@ -22,6 +24,15 @@ var failingObj = &example.Pod{TypeMeta: metav1.TypeMeta{Kind: "foo"}, ObjectMeta
 var exampleList = &example.PodList{TypeMeta: metav1.TypeMeta{Kind: "foo"}, ListMeta: metav1.ListMeta{}, Items: []example.Pod{*exampleObj}}
 var anotherList = &example.PodList{Items: []example.Pod{*anotherObj}}
 
+func newTestMode1(t *testing.T, ls LegacyStorage, us Storage) DualWriter {
```

Comment:
Added this and the one for Mode2 as well since it's a bit noisy on the terminal.